### PR TITLE
[Feature] : Our services section enhancement

### PR DESCRIPTION
**Pull request by SahilVermaFED**
## Description

-   [Ticket Link]
-  our services section fully responsive with scrolling and line background design

## UI screenshot or Video
[Screencast from 13-03-24 12:51:08 PM IST.webm](https://github.com/primathonacademy/april-batch-live-project/assets/131689244/15e034cf-2f73-46fd-bdba-6d1865f2507a)
[Uploading Screencast from 13-03-24 12:51:08 PM IST.webm…]()

### General:

-   [x] **PR Size :** Confirming PR size < 20 files or pre-approved if > 20 files.
-   [x] **Self Review :** Confirming I have self-reviewed my PR
-   [x] **Peers Review :** - [ ] @Dev 1
-   [x] Confirming I have added all sensitive information-related should be inside .env files

### Client:

-   [x] **Console :** Confirming no console errors in the UI for this update.
-   [x] **Import order :** Confirming correct import order.

```
# External Import
import React, { useState, useEffect } from 'react';
import { useParams } from 'react-router-dom';
import { useDispatch, useSelector, shallowEqual } from 'react-redux';

# Internal Import
import { } from '../../Api/ApiRoutes';
import { } from '../../constants/AppConstant';
```

-   [x] **Re-renders :** Confirming minimal re-renders for new React components.

-   [x] **Network calls :** Confirming efficient network/API calls, minimized as needed.

-   [x] **Util function comments :** Confirming comprehensive comments for utility functions.

-   [x] **Image optimization :** Confirming I have checked that all images are optimized. If not, please use https://tinyjpg.com/ or https://vecta.io/nano.

-   [x] **Dead code :** Confirming I have removed all comments code which is of no use OR written why I have kept the commented code.

### Confirming different viewport testing of the entire flow:

-   [x] Tested on 280px viewport
-   [x] Tested on 320px viewport
-   [x] Tested on 375px viewport
-   [x] Tested on 425px viewport
-   [x] Tested on 768px viewport
-   [x] Tested on 1024px viewport
-   [x] Tested on 1440px viewport
-   [x] Tested on 2560px viewport
### End-to-end testing:

-   [x] **Happy flow :** Successfully tested end-to-end happy flow, all working as expected.
-   [x] **Error flow :** Tested UI flow with error scenarios, gracefully handled errors on UI
-   [x] **Common component changed :** Tested change in common component across views to prevent breaking elsewhere.

### CODE REVIEW

File: src/pages/homepage/OurServiceSection.jsx [hunk 1]
```diff
@@ -1,135 +1,145 @@
...
-            );
+    const rawText =
+        'Yume Labs is a global UI UX Design and Industrial Design agency that helps your business scale through fail-proof design innovation.';
 
-            return () => {
-                pin.kill();
-            };
-        }
+    let displayText = rawText;
+    if (window.innerWidth >= SCREEN_SIZE.MD && window.innerWidth <= SCREEN_SIZE.LG) {
```
Comment (ram-primathon) [inline src/pages/homepage/OurServiceSection.jsx]: use css for this

**Review by ram-primathon — CHANGES_REQUESTED** [review]

File: src/pages/homepage/OurServiceSection.jsx (hunk 1 above)
Comment (ram-primathon) [reply src/pages/homepage/OurServiceSection.jsx]: display: -webkit-box;
-webkit-box-orient: vertical;
-webkit-line-clamp: 3;
overflow: hidden;

File: tailwind.config.js [hunk 2]
```diff
@@ -90,5 +90,6 @@ export default {
             }
         }
     },
-    plugins: []
+    // eslint-disable-next-line no-undef
```
Comment (ram-primathon) [inline tailwind.config.js]: this should be the last choice

**Review by ram-primathon — CHANGES_REQUESTED** [review]

File: src/pages/homepage/OurServiceSection.jsx (hunk 1 above)
Comment (TheSahilVerma) [reply src/pages/homepage/OurServiceSection.jsx]: done Sir

File: tailwind.config.js (hunk 2 above)
Comment (TheSahilVerma) [reply tailwind.config.js]: done sir

**Pull request merged**